src/main/java/org/jabref/gui/ai/components/summary/SummaryShowingComponent.java

```diff
@@ -46,6 +60,42 @@ private static String formatTimestamp(LocalDateTime timestamp) {
         return timestamp.format(DateTimeFormatter.ofLocalizedDateTime(FormatStyle.MEDIUM).withLocale(Locale.getDefault()));
     }
 
+    private void initializeMarkdownWebView() {
+        if (markdownWebView == null) {
+            markdownWebView = new WebView();
+            VBox.setVgrow(markdownWebView, Priority.ALWAYS);
+
+            markdownWebView.prefWidthProperty().bind(summaryTextArea.widthProperty());
```

Comment:
Hmm, typically we don't set the preferred width. We only set paddings, spacing, and VGrow or HGrow.

I think this is related to my idea of using single `WebView` in final review comment. If you use single `WebView`, then you shouldn't have those problems

Reply:
I had to use minWidth otherwise, it would generate the `button` and the `checkbox`, and to make it appear user would need to drag the editor up.